validator/keymanager/remote-web3signer/internal/client.go

```diff
@@ -116,27 +120,35 @@ func (client *ApiClient) GetServerStatus(ctx context.Context) (string, error) {
 
 // doRequest is a utility method for requests.
 func (client *ApiClient) doRequest(ctx context.Context, httpMethod, fullPath string, body io.Reader) (*http.Response, error) {
+	ctx, span := trace.StartSpan(ctx, "remote_web3signer.client.doRequest")
+	defer span.End()
+	span.AddAttributes(
+		trace.StringAttribute("httpMethod", httpMethod),
+		trace.StringAttribute("fullPath", fullPath),
+		trace.StringAttribute("hasBody", fmt.Sprintf("%v", body != nil)),
```

Comment:
Use BoolAttribute https://pkg.go.dev/go.opencensus.io/trace?utm_source=godoc#BoolAttribute

```suggestion
		trace.BoolAttribute("hasBody", body != nil),
```

Reply:
good catch